tools/c7n_index/c7n_index/metrics.py

```diff
@@ -229,6 +278,55 @@ def index_account(config, idx_name, region, account, start, end, period):
     return region_time, region_points
 
 
+def index_account_resources(config, account, region, policy, date):
+    indexer = get_indexer(config, type=policy['resource'])
+
+    s3 = local_session(
+        lambda : assumed_session(account['role'], 'PolicyIndex')).client('s3')
+
+    bucket = account['bucket']
+    key_prefix = "accounts/" + account['name'] + "/" + region + \
```

Comment:
minor but stylistic pref for string formatting via "accounts/%s/%s" % (account['name'], region). also most folks use account id for archive as its not mutable, worth looking at making they account attribute output prefix configurable.